src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -1103,7 +1103,7 @@ static TiebreakMode getByInternalLabel(String labelLookup) {
       return Arrays.stream(TiebreakMode.values())
           .filter(v -> v.internalLabel.equals(labelLookup))
           .findAny()
-          .orElse(null);
```

Comment:
Nit: this might be a good time to name these in a more clear fashion: `OVERVOTE_RULE_UNKNOWN`, `TIEBREAK_MODE_UNKNOWN`, `WINNER_ELECTION_MODE_UNKNOWN`.

Reply:
Eh, let's live dangerously.